packages/x-data-grid-premium/src/hooks/features/chartsIntegration/useGridChartsIntegration.tsx

```diff
@@ -156,6 +156,7 @@ export const useGridChartsIntegration = (
     | 'experimentalFeatures'
   >,
 ) => {
+  const activeChartId = React.useRef<string>(gridChartsIntegrationActiveChartIdSelector(apiRef));
```

Comment:
Wouldn't `useGridSelector()` reactive binding be better suited here to keep up-to-date with the latest value from `gridChartsIntegrationActiveChartIdSelector`?